authorization/src/main/java/uk/gov/di/ipv/cri/common/api/handler/AuthorizationHandler.java

```diff
@@ -70,6 +72,12 @@ public APIGatewayProxyResponseEvent handleRequest(
             authorizationValidatorService.validate(authenticationRequest, sessionItem);
 
             // create authorization
+            if (sessionItem.getAuthorizationCode() == null
```

Comment:
There is no need for the null check here as it happens inside `StringUtils.isBlank` method logic